examples/sampling/graphbolt/node_classification.py

```diff
@@ -261,27 +308,19 @@ def parse_args():
         help="Fan-out of neighbor sampling. It is IMPORTANT to keep len(fanout)"
         " identical with the number of layers in your model. Default: 15,10,5",
     )
-    parser.add_argument(
```

Comment:
Why we do need to remove the `--device` option?

Reply:
Current version is a cpu-only one. I was planning to create another PR to complete the cuda version.